scripts/nsls2-dev-build.sh

```diff
@@ -20,7 +20,7 @@ docker pull $IMAGE_NAME
 
 echo "Running the docker container"
 
-cat << EOF | docker run -i \
+cat << EOF | docker run -i --rm\
```

Comment:
I would add an extra blank space just in case. Same for the below.

Reply:
good catch.